microsoft/vscode-powerquery
# More UI test improvements

- Use `vscode-test` script everywhere
- Run UI tests with `--temp-profile` to ensure that there is no state corruption from local environment
- Add tests for external symbol modules
- Add delay to diagnostics tests

Diagnostics were flaky when run multiple times locally. Adding a small delay (`10ms`) greatly improves the behavior.
